### PR TITLE
Added a document on NSG log shipping

### PR DESCRIPTION
**Pull request by nico-shishkin**
# What changed

https://deploy-preview-1234--logz-docs.netlify.app/shipping/log-sources/azure-nsg.html

----

## Pages to review

- LinkToPage1
- LinkToPage2

## Remaining work

- [ ] Technical review
- [ ] Copy Review
- [ ] Redirects: 

## Post launch

To be completed by the docs team upon merge:

- [ ] Teams to update with the new information:
- [ ] Replace original content on the support portal with 'this page has been moved to ...' - paste the URL
- [ ] Update these log shipping pages in the app: - paste the URL

### CODE REVIEW

File: _source/logzio_collections/_log-sources/azure-nsg.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+1. For your VM, navigate to **Networking > NSG > NSG flow**.
+2. From the list of NSGs, select the NSG with the name of your VM.
+3. Set the **Flow logs** status to **on**.
+4. Select the required **Flow logs version**.
+5. In the **Storage accound** field, select the Azure Blob Storage account for this resource group.
+6. Select the required retention period.
+7. If required, enable the **Traffic Analytics**.
+8. Save the configuration.
+
```
Comment (Doron-Bargo) [inline _source/logzio_collections/_log-sources/azure-nsg.md:43]: Please add a step to keep the storage name for the blob trigger step

File: _source/logzio_collections/_log-sources/azure-nsg.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+##### Connect your Azure Blob Storage account to Logz.io
+
+Connect your Azure Blob Storage account to Logz.io as described in the instructions [here](https://docs.logz.io/shipping/log-sources/azure-blob.html#existing-blob-config).
```
Comment (Doron-Bargo) [inline _source/logzio_collections/_log-sources/azure-nsg.md:47]: please split https://docs.logz.io/shipping/log-sources/azure-blob.html#existing-blob-config to something generic that we can add here

**Review by Doron-Bargo — CHANGES_REQUESTED** [review]
Hi, please see my comments

File: _source/logzio_collections/_log-sources/azure-nsg.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+<div class="tasklist">
+
+##### Register Insights provider
+
+Register the Microsoft.Insights provider as described [here](https://docs.microsoft.com/en-us/azure/network-watcher/network-watcher-nsg-flow-logging-portal#register-insights-provider).
+
+##### Enable NSG flow log
+
+1. For your VM, navigate to **Networking > NSG > NSG flow**.
```
[the remark this answers is not in the document]
Comment (YajB) [reply _source/logzio_collections/_log-sources/azure-nsg.md]: > We already have numbers on the title so maybe use letters here?

I disagree.  It is cognitively easier to process numbered steps in a procedure. 
Those titles are for procedures.

**Review by YajB — APPROVED** [review]
Resolved the issue of using numbers for substeps in higher level sections that are also sections

**Pull request merged**